### DISCUSSION TITLE
Alert and Reporting Not able to Take SnapShots

### DISCUSSION THREAD

**Discussion by codegeekjk** [apache/superset#15667] [Q&A / Help]
I am getting the below error while taking SnapShot :

`Error: Failed taking a screenshot Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//0.0.0.0%3A8088/login/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%200.0.0.0%3A8088.`

In SuperSet Docker how to I autheticate my WebDriver ? Any help

**Comment by nytai** [comment]
looks like a network error, are you sure superset is available at `0.0.0.0:8088` in the internal docker network? If you're using docker-compose you can try specifying the service name instead, eg `superset:8088`
